Updated BCM gains and offsets

woltzoe opened the pull request:
Updated BCM gains and offsets with the constant fit values. Also added saturation information to both files.

Commit:
Updated BCM gains and offsets

Commit:
Calibration files for BCM

ccotton98 commented:
Good work! For which runs should the period1 file be used vs. the period2 file? Could this information be added in the comments of the param files? eg. "this calibration should be used for SHMS runs 1100-1300 and HMS run 350-900". Could you also add comment specifying which BCMs were "good" (1,2 and 4A). Finally, could you also update the shms/hms standard.database files so that the correct BCM calibration is pointed to for a given run. We can discuss how best to organize this in standard.database if it isn't clear.

Commit:
Modified the files to include which BCMs should be used and to note that either gbcm_period1_fa22.param or gbcm_period2_fa22.param file can be used for any run, because they are now the same.

Commit:
gbcm_period1_fa22.param and gbcm_period2_fa22.param parameter files were the same file. Deleted gbcm_period1_fa22.param file and renamed the name of gbcm_period2_fa22.param to gbcm_fa22.param.

gbcm_period1_fa22.param is not found anywhere
gbcm_period2_fa22.param is found in DBASE/(S)HMS/standard.database files. Parameter files there are renamed gbcm_fa22.param

Commit:
Forgot to add gbcm_fa22.param file. gbcm_period2_fa22.param was renamed to gbcm_fa22.param
